File: Android/Sources/UserControlLibraries/GoogleMapsLib/src/main/java/com/genexus/android/controls/maps/google/MapRouteLayer.java
package com.genexus.android.controls.maps.google;

import android.app.Activity;

import com.genexus.android.maps.DirectionsServiceRequest;
import com.genexus.android.core.controls.maps.common.IGxMapViewSupportLayers;

import static com.genexus.android.controls.maps.google.MapsProvider.PROVIDER_ID_DIRECTIONS;

public class MapRouteLayer extends DirectionsServiceRequest<MapLocation, MapLocationBounds, GxMapViewItem, GxMapViewData> {
	public MapRouteLayer(IGxMapViewSupportLayers gxMapView, Activity activity) {
		super(gxMapView, activity, null, PROVIDER_ID_DIRECTIONS);
	}
}

File: Android/Sources/UserControlLibraries/BaiduMaps/src/main/java/com/genexus/android/controls/maps/baidu/MapRouteLayer.kt
package com.genexus.android.controls.maps.baidu

import android.app.Activity
import com.genexus.android.controls.maps.baidu.MapsProvider.PROVIDER_ID_DIRECTIONS
import com.genexus.android.core.controls.maps.common.IGxMapViewSupportLayers
import com.genexus.android.maps.DirectionsServiceRequest

internal class MapRouteLayer(mapView: IGxMapViewSupportLayers?, activity: Activity) :
    DirectionsServiceRequest<MapLocation, MapLocationBounds, GxMapViewItem, GxMapViewData>
    (mapView, activity, null, PROVIDER_ID_DIRECTIONS)
